site/gatsby-site/cypress/e2e/integration/submit.cy.js

```diff
@@ -1546,4 +1581,102 @@ describe('The Submit form', () => {
 
     cy.get('@parseNewsSpy').should('have.been.calledOnce');
   });
+
+  it('Should load from localstorage', () => {
```

Comment:
It would be helpful to have an additional test that simulates form filling and verifies that the `localStorage` data was correctly saved.

Reply:
example here: https://github.com/responsible-ai-collaborative/aiid/pull/2007/files#diff-8bc19c4a77b6eaa334aa3a8cf9e1f4d0a055a3762ed36db56128bf073c9542b4R32